storyStarterGame.js

```diff
@@ -0,0 +1,470 @@
+const readLineSync = require('readline-sync');
+
+    
+const welcome = () => {
+    let nameInput = readLineSync.question("What is your name?") // Ask the player what is their name. 
+
+console.log (`Hello ${nameInput}!  It's nice to meet you.`) // Using the information gotten above, this line prints a greeting with the player's name 
+console.log ("Welcome to Story Generator Game!") // Welcoming the player to the game
+    console.log ("Where I will help you write the next best seller!") // Stating the purpose of the game
+    if (readLineSync.keyInYN("Are you ready to begin?")) { // This line ask asks the player if they are ready to begin the game. 
+        console.clear() // This clears all of the previous lines 
+        startGame(); // If the player presses Y then the game will go straight to this function, starting the game!
+    } else {
+        leaveGame(); // If the player presses N then the game will go straight to this function, ending the game!
+    }
+}
+
+
+    // Starting Story Game 
+const startGame = () => {
+    console.log ("There are two genres we will be working with today") // Letting the player know that there are going to be two chocies here. 
+    let choice = readLineSync.question("The choices are Romance and Suspense! Which would you like to choose to be your genre?") // Asking whether or not the player wants a romance story or a suspense story
+    if (choice === "Suspense" || choice === "suspense") { // if the choice that was made equal to the input "Suspense" 
+        console.clear(); // Clears all of the previous lines 
+        suspenseStory(); // If player's input equals to "suspense" than it will run into the next function.
+    } else if (choice === "Romance" || choice === "romance"){ // if the choice that was made equal to "romance"
+        romanceStory(); // If the player's input equals to "romance" than it will run into the next funtion 
+    } else {
+        leaveGame(); // If the player's input is anything else then it will leave the game. 
+    }
+} 
+
+
+// suspenseStory
+const suspenseStory = () => {
+    console.log("Now with any good story there are great characters to tell their stories.")
+    let characterNum = readLineSync.question ("How many character you want in this story?")
+    if (characterNum > 3) {
+        tooManyChar();
+    } else {
+        perfectNum();
+    }
+} 
+
+//perfectNum
+const perfectNum = () => {
+    console.log ("Perfect!!!")
+    console.log ("You will now have to choose your storyline!")
+    let setting = readLineSync.question ("Do you want to start your story in the dark woods or on the dark road?")
+    if (setting === "dark woods") {
+        console.clear();
+        darkWoodsStoryLine();
+    } else {
+        darkRoadStoryLine();
+    }
```

Comment:
What if the user had entered a incorrect answer to this question ? What if the user typed in `Dark Woods` ?

Reply:
I am still trying to figure this one out because it shouldn't matter as long as the got the spelling correctly. So would I use .toLowerCase() and .toUpperCase()